src/crystal/system/win32/process.cr

```diff
@@ -103,7 +103,7 @@ struct Crystal::System::Process
   def self.on_interrupt(&@@interrupt_handler : ->) : Nil
     restore_interrupts!
     @@win32_interrupt_handler = handler = LibC::PHANDLER_ROUTINE.new do |event_type|
-      next 0 unless event_type.in?(LibC::CTRL_C_EVENT, LibC::CTRL_BREAK_EVENT)
+      next 0 unless event_type.in?(LibC::CTRL_C_EVENT, LibC::CTRL_BREAK_EVENT, LibC::CTRL_CLOSE_EVENT, LibC::CTRL_LOGOFF_EVENT, LibC::CTRL_SHUTDOWN_EVENT)
```

Comment:
I'd suggest moving these into a constant for readability purposes.

Reply:
We could probably remove the line entirely as I'm pretty sure these are all the possible events